index.js

```diff
@@ -75,7 +75,7 @@ class Sender {
   }
 
   async _send(message, callback) {
-    var buf = Buffer.from(message.replace(/\s+$/, "") + os.EOL);
```

Comment:
we can't have trailing whitespaces here as  the message here is a result of `JSON.stringify`